src/testing/time_refinement_study_advection.cpp

```diff
@@ -0,0 +1,135 @@
+#include "time_refinement_study_advection.h"
+#include "flow_solver.h"
+#include "flow_solver_cases/periodic_1D_flow.h"
+#include "cmath"
+
+namespace PHiLiP {
+namespace Tests {
+
+template <int dim, int nstate>
+TimeRefinementStudyAdvection<dim, nstate>::TimeRefinementStudyAdvection(
```

Comment:
This class looks quite general in terms of handling any PDE type, if thats the case, it could be renamed `TimeRefinementStudy` or `TemporalConvergenceStudy`?

Reply:
Yes, it's general as long as there is an analytical solution available. Changing in the next commit to `TimeRefinementStudy`.